geo-types/src/line_string.rs

```diff
@@ -132,19 +138,52 @@ impl<'a, T: CoordNum> DoubleEndedIterator for PointsIter<'a, T> {
     }
 }
 
+/// A [Coordinate] iterator used by the `into_iter` method on a [LineString]
+#[derive(Debug)]
+pub struct CoordinatesIter<'a, T: CoordNum + 'a>(::std::slice::Iter<'a, Coordinate<T>>);
```

Comment:
Not blocking, but this could also implement [`ExactSizeIterator`](https://doc.rust-lang.org/std/iter/trait.ExactSizeIterator.html)

Reply:
Done!